util/test/cardano-sl-util-test.cabal

```diff
@@ -16,27 +16,38 @@ library
   exposed-modules:
                        Test.Pos.Util.Base16
                        Test.Pos.Util.Gen
+                       Test.Pos.Util.Golden
                        Test.Pos.Util.Modifier
                        Test.Pos.Util.Orphans
                        Test.Pos.Util.QuickCheck
                        Test.Pos.Util.QuickCheck.Property
                        Test.Pos.Util.QuickCheck.Arbitrary
+                       Test.Pos.Util.Tripping
```

Comment:
While not strictly necessary, since only `core` & `crypto` seem to import these (and they are "above" `binary`), I think it's good to have these in `util` 👍

Reply:
This is not a required change... not sure how to indicate that.

Reply:
You did indicate that! That's what :+1: or :100: means :)